test/e2e/e2e.sh

```diff
@@ -40,7 +40,7 @@ main() {
 
     # Build static binaries; otherwise go test complains.
     export CGO_ENABLED=0
-    go test -v -count=1 --tags=e2e ${REPO}/test/e2e/...
+    go test -v -count=1 --tags=e2e ${REPO}/test/e2e/e2e_test.go
```

Comment:
I have changed the behaviour to include all package in `e2e` directory. 

Initially this was done because the test was failing as the `client` directory under `e2e` doesn't have any `_test` files. Now I have just excluded the `client` directory from go test, while still building the files in the `client` directory.